### ISSUE TITLE
0.5.2 svd `lib` unexpected

### ISSUE THREAD

**Issue by 1Reinier** [lebedov/scikit-cuda#154]
Hi there,
Thanks for all the great work!
I'm trying to use one of the new CUSOLVER functions in 0.5.2, but I'm running into an error that `lib` is unexpected. I checked my `skcuda.__version__` and it really is 0.5.2.
```
    U, s, V = skcuda.linalg.svd(X, lib='cusolver')
TypeError: svd() got an unexpected keyword argument 'lib'
```
Can anybody help me figuring out what went wrong? I'm running on a tk1 so I can't revert to CULA.
Thanks!
Reinier

**Comment by lebedov** [comment]
Seems like an installation issue. Can you check whether the installed `skcuda/linalg.py` actually contains the latest version of the `svd()` function? If not, try reinstalling.

**Comment by 1Reinier** [comment]
I was running two installations at the same time, so problem resolved.
One last question: do you know if cuSOLVER is available for the arm platform? I'm running on a Jetson TK1.
Thanks!

**Comment by lebedov** [comment]
Great. 

Although CUDA 6.5 [appears to be available](https://developer.nvidia.com/cuda-toolkit-65) for Linux on ARM, the patch that NVIDIA prepared to make cuSOLVER available for CUDA 6.5 only appears to be for x86. I haven't seen anything about CUDA 7+ on ARM.
